content/artwork/index.html.haml

```diff
@@ -1,13 +1,57 @@
 ---
-layout: default
+layout: simplepage
 title: Jenkins Artwork
+tight: true
+notitle: true
 ---
 
-/ Please add new logo meta-data to content/images/logos and content/_data/logo
+:css
+  .artwork-container {
+    padding: 2rem 0;
+  }
+  .logo-thumb {
+    max-width: 100%;
+    height: auto;
+    border-radius: 8px;
+    margin-bottom: 1rem;
+  }
+  .logo-card {
+    text-align: center;
+    padding: 1rem;
+    background: var(--background);
+    border: 1px solid var(--border-color);
+    border-radius: 8px;
+    margin-bottom: 1.5rem;
+    transition: 0.2s ease;
+    &:hover {
```

Comment:
The click effect is a little heavy, it also displays when you click a link whereas it should probably only show when you click the image.

<img width="251" alt="image" src="https://github.com/user-attachments/assets/af36135e-88a5-4f6a-b3c4-2c16864d2753" />